### DISCUSSION TITLE
Can i use django-vectordb to store vectors that is already embedded by OpenAI from documents/pdf?

### DISCUSSION THREAD

**Discussion by wzharith** [pkavumba/django-vectordb#13] [Q&A]
I am a very much newbie in this area. And come across this repo which is the most similar to what i want to use. But i see there are only 2 types of insertion: 
1. adding model instances
2. adding text to model.

so if i have like a already converted vectors, can i import and store it using django-vectordb?
I am currently stuck because i dont know what the step is after this:

```python 
pages = loader.load_and_split()
db = FAISS.from_documents(pages, OpenAIEmbeddings()) 
retriever = db.as_retriever(search_kwargs={'k': 3, 'fetch_k': 100}) # Persist the vectors locally on disk db.save_local("db_faiss_amended_2_pdf") # this will store in a folder
faiss = FAISS.load_local("db_faiss_amended_2_pdf", OpenAIEmbeddings()) #read from folder
```

In this example i use faisss and store it locally. but now i want to use this django-vectordb. so i can have a proper database and not locally.

**Comment by pkavumba** [comment]
Hi! These two methods are just abstractions. You can still interact with the database just like any other db in Django. `get_vectordb_model()` will give you the actual model. 

From another issue, we working on adding langchang as_retriever support
